tests/test_cli.py

```diff
@@ -196,6 +126,17 @@ def test_tc_help(record_property, capsys) -> None:
     assert pytest_wrapped_e.value.code == 0, "ExitCode not as expected."
 
 
+@pytest.fixture(autouse=True)
+def _setup_and_teardown():
+    # lobster-exclude: This is a simple helper function that prepares and cleanup the tests.
+    """Before running the test, delete the LOBSTER file if it exists."""
+    # Preparation:
+    if Path(TEST_LOBSTER_OUTPUT_FILE).exists() and Path(TEST_LOBSTER_OUTPUT_FILE).is_file():
+        Path(TEST_LOBSTER_OUTPUT_FILE).unlink()
+    yield
+    # Teardown:
```

Comment:
There are still 2 issues with doing it this way.

1. The output from the last test is left over as only the setup phase removes the generated file.
2. The file name is static, breaking parallel test execution that eventually are used in CI systems (see pytest-xdist).

Check https://docs.pytest.org/en/stable/how-to/tmp_path.html how this can be improved.